lesson2/argirovatina_homework_2.rb

```diff
@@ -0,0 +1,35 @@
+module M
+
+  def self.included(base)
+    base.extend(C)
+  end
+
+  def custom_instance_method
+    p Foo.instance_methods - Object.instance_methods
```

Comment:
Not bad, but that's not what I asked about.
You need to output to the console when **you are including methods, not in methods itself**.